docs/jwst.rst

```diff
@@ -162,6 +167,13 @@ can still be used, but the same masks can also be referred to as "MASKRND" and
 "MASKSWB" or "MASKLWB", the nomenclature that was eventually adopted for use in
 APT and other JWST documentation. Both ways work and will continue to do so.
 
+The NIRCam class can automatically switch its ``aperturename`` attribute when a
+coronagraphic mask is selected, to select the aperturename for the appropriate
+coronagraphic subarray.  The detector reference pixel location will also update
+to the center of the coronagraphic subarray. This behavior on image mask or
+pupil mask selection can be disabled by setting ``nircam.auto_aperturename =
+False``.
+
```

Comment:
Should also mention that the aperture name will always change if the `detector` property (attribute?) changes.